app/Mails/SendIntegrationActivationReminderEmail.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Mails;
+
+use App\Domain\Contacts\Contact;
+use App\Domain\Integrations\Events\ActivationReminderEmailSend;
+use App\Domain\Integrations\Integration;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Domain\Mail\MailManager;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+use Psr\Log\LoggerInterface;
+
+final readonly class SendIntegrationActivationReminderEmail
+{
+    public function __construct(
+        private MailManager $mailManager,
+        private IntegrationRepository $integrationRepository,
+        private LoggerInterface $logger
+    ) {
+    }
+
+    public function send(Collection $integrations): void
+    {
+        foreach ($integrations as $integration) {
+            $this->mailManager->sendActivationReminderEmail(new ActivationReminderEmailSend($integration->id));
```

Comment:
This is not how `Events` are supposed to be handled. They should be dispatched and the listener will be called by the framework, not with direct method calls.